### DISCUSSION TITLE
Self-Hosted - Change default feed to a custom feed?

### DISCUSSION THREAD

**Discussion by RandomNinjaAtk** [webrcade/webrcade#110] [Q&A]
So after poking around and finding the docker through various posted issue and how to map the local volume for files /content. I have everything working. However, by default when you visit the site, it loads the default feed provided. I can add the local feed after the site loads using the web UI manually, however, I would like to just have it load the custom feed automatically and not require any user input, is this possible?

**Comment by avinceslao** [comment]
I 2nd this. Is there any way to do this? I love this project everyone. Kudos to the dev. Please setup a patreon so i can donate!

**Comment by navarrofnc** [comment]
Same question here.
